packages/web-sockets/src/utils/constants.js

```diff
@@ -0,0 +1,8 @@
+const supportedEnvs = {
+  PRODUCTION: 'production',
+  DEVELOP: 'develop',
```

Comment:
can we change this to `development`?
```suggestion
  DEVELOPMENT: 'development',
```